oci/main.py

```diff
@@ -26,21 +29,80 @@ def run():
         'change',
         help='Gerrit change number')
 
+    system_tests_parser = subparsers.add_parser(
+        "system-tests",
+        help="system-tests for a change")
+    system_tests_parser.set_defaults(command=system_tests)
+
+    system_tests_parser.add_argument(
+        'change',
+        help='Gerrit change number')
+
+    system_tests_parser.add_argument(
+        '--engine_version',
+        default='master')
```

Comment:
We need giving some example of possible values e.g. "4.3", and pointing to the place listing all options.

Reply:
Should i write that in the help?
Something along the lines off:
"Engine version -- e.g., 'master', '4.3' etc. Full list of versions can be seen here https://blahblah"
?
Also, I am not sure where can i get the versions from, the release notes? https://www.ovirt.org/release/

Reply:
Done.